CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+<!-- TABLE OF CONTENTS -->
+
+## Table of Contents
+
+- [About the Project](#about-the-project)
+  - [Built With](#built-with)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+[Docusaurus](https://docusaurus.io/) is a static site generator that helps you ship beautiful, accessible docs. For building our documentation engine, we have made certain modifications to the template generated by [Docusaurus](https://docusaurus.io) to be able to properly showcase all our contents with their respective versions.
+
+### Built With
+
+- [Docusaurus](https://docusaurus.io/)
+- [React](https://reactjs.org/)
+- [Tailwind](https://tailwindcss.com/)
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+This section describes how you can get our documentation portal up and running on your machine.
+
+### Prerequisites
+
+- [node](https://nodejs.org/en/)
+- [npm](https://www.npmjs.com/)
+
+### Installation
+
+1. Clone the repo
+
+```shell
+git clone https://github.com/keptn-sandbox/new-keptn-docs-engine
+```
+
+2. Install NPM packages
+
+```shell
+npm install
+```
+
+3. Run the app
+
+```shell
+npm start
+```
+
+<!-- USAGE EXAMPLES -->
+
+## Usage
+
+<!-- In usage, mention how to edit the docs, how to update versions, etc. -->
+
+### Writing Documentation
+
+To just edit older documentation, go to the specified versioned folder for a section, for example, you want to edit documentation for Release 0.16.x, open up `Release 0.16.x` folder and edit the required files there.
+
+To create a new version inside a section, for example, Go. Make your changes in [docs/Go](./docs/Go). The `./docs` folder consists of the `next` version, which is unpublished, and is where you add your newer or _next_ version of the documentation.
+
+After your changes are done, to create a version, run the following command:
+
+```shell
+npm run docusaurus docs:version:Go 1.2.3
+```
+
+This will create a new version `1.2.3` for Go.
+
+Reference: [https://docusaurus.io/docs/versioning](https://docusaurus.io/docs/versioning)
+
+### To add new sections
```

Comment:
I guess I need to know what you are defining as a section.  This just seems like a whole lot of overhead for writers to go through to add a section.

Reply:
Section means a new doc that will be added in the sidebar of the site.